data/pathfinder/paizo/roleplaying_game/ultimate_magic/um_abilities_class.lst

```diff
@@ -1456,7 +1456,7 @@ Tiring Critical			KEY:Ki Power ~ Tiring Critical		CATEGORY:Special Ability	TYPE:
-Possessed Oracle		KEY:Oracle Archetype ~ Possessed Oracle		CATEGORY:Archetype	TYPE:Archetype.OracleArchetype.OracleMysterySpell2.OracleMysterySpell4.OracleMysterySpell6.OracleMysterySpell8.OracleMysterySpell10.OracleMysterySpell12.OracleMysterySpell16.OracleRevelation1												PREABILITY:1,CATEGORY=Special Ability,Oracle ~ Haunted,Oracle ~ Tongues					PRECLASS:1,Oracle=1	PREMULT:1,[PREABILITY:1,CATEGORY=Archetype,Oracle Archetype ~ Possessed Oracle],[!PREABILITY:1,CATEGORY=Archetype,TYPE.OracleMysterySpell2,TYPE.OracleMysterySpell4,TYPE.OracleMysterySpell6,TYPE.OracleMysterySpell8,TYPE.OracleMysterySpell10,TYPE.OracleMysterySpell12,TYPE.OracleMysterySpell16,TYPE.OracleRevelation1]															DESC:Some oracles are possessed by spirits, demons, or similar beings.																ABILITY:Internal|AUTOMATIC|Possessed Oracle ~ Bonus Spells|PRECLASS:1,Oracle=1	ABILITY:Oracle Class Feature|AUTOMATIC|Possessed Oracle ~ Two Minds|PRECLASS:1,Oracle=1																																			BONUS:VAR|OracleMysteryLevel|-1|PRECLASS:1,Oracle=1										COST:0	SOURCEPAGE:p.59	ASPECT:Archetype Base Class|Oracle
+Possessed Oracle		KEY:Oracle Archetype ~ Possessed Oracle		CATEGORY:Archetype	TYPE:Archetype.OracleArchetype.OracleMysterySpell2.OracleMysterySpell4.OracleMysterySpell6.OracleMysterySpell8.OracleMysterySpell10.OracleMysterySpell12.OracleMysterySpell16.OracleRevelation1												PREABILITY:1,CATEGORY=Special Ability,Oracle ~ Haunted,Oracle ~ Tongues,Oracle ~ Pranked		PRECLASS:1,Oracle=1	PREMULT:1,[PREABILITY:1,CATEGORY=Archetype,Oracle Archetype ~ Possessed Oracle],[!PREABILITY:1,CATEGORY=Archetype,TYPE.OracleMysterySpell2,TYPE.OracleMysterySpell4,TYPE.OracleMysterySpell6,TYPE.OracleMysterySpell8,TYPE.OracleMysterySpell10,TYPE.OracleMysterySpell12,TYPE.OracleMysterySpell16,TYPE.OracleRevelation1]															DESC:Some oracles are possessed by spirits, demons, or similar beings.																ABILITY:Internal|AUTOMATIC|Possessed Oracle ~ Bonus Spells|PRECLASS:1,Oracle=1	ABILITY:Oracle Class Feature|AUTOMATIC|Possessed Oracle ~ Two Minds|PRECLASS:1,Oracle=1																																			BONUS:VAR|OracleMysteryLevel|-1|PRECLASS:1,Oracle=1										COST:0	SOURCEPAGE:p.59	ASPECT:Archetype Base Class|Oracle
```

Comment:
I've avoided Oracles as long as I could, but man, these guys need a good overhaul... Why do the Oracle Archetypes require "x" class features? No other Archetype follows this pattern. Anyways, other than the fact this is approved.